google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentConfigurationPanel.java

```diff
@@ -123,6 +128,12 @@ public void applyEditorTo(@NotNull AppEngineDeploymentConfiguration configuratio
     configuration.setPromote(promoteCheckbox.isSelected());
     configuration.setCloudProjectName(projectSelector.getText());
     configuration.setStopPreviousVersion(stopPreviousVersionCheckbox.isSelected());
+    configuration.setDeployAllConfigs(deployAllConfigsCheckbox.isSelected());
```

Comment:
oh I see - the flex editor is explicitly hiding it; but since its standard only config (for now) shouldn't this live in the standard editor?